packages/web-app-admin-settings/src/components/Spaces/SideBar/MembersRoleSection.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div>
```

Comment:
What do you think about making this a `ul` with `li`s? We're listing members... so a list type would be appropriate.

Reply:
Good point, makes sense!